cli/src/commands/extensions/mod.rs

```diff
@@ -114,6 +125,35 @@ pub async fn handle_run_extension(
     Ok(CommandValue::Code(ExitCode::Ok))
 }
 
+/// Handle the `extension run <PATH>` command path.
+///
+/// Run the extension located at the given path.
+pub async fn handle_run_local_extension(
```

Comment:
I don't really like the name of `run_local_extension`, kinda makes it sound like there's a way to run remote extensions too.

Reply:
Changed it to `handle_run_extension_from_path`.